simpledotorg/simple-server
# 2 million patients registers in India celebration card on dashboard homepage

**Story card:** [sc-8152](https://app.shortcut.com/simpledotorg/story/8152/implement-feature-in-dev-environment)

## Because

Completed celebration section

## This addresses

Card on dashboard celebrating 2million registered users in India.
Only visible in India.

## Test instructions

- Check responsiveness of design
- Check link correctly loads social media image
- Check social media image for correct language and typos
- Check that design is not terrible on IE
- Check only visible to India
